modules/appeals_api/app/models/appeals_api/claimant.rb

```diff
@@ -0,0 +1,117 @@
+class Claimant
+  def initialize(auth_headers, form_data)
+    @auth_headers = auth_headers
+    @form_data = form_data
```

Comment:
I'm not sure this needs the full form data since it's only concerned about the claimant fields. I think passing in `form_data['data']['attributes']['claimant']` to the initializer would be fine, unless I'm missing something?

Reply:
totes right